cms/templates/cms/home_page.html

```diff
@@ -45,7 +52,7 @@
       <div class="banner-wrapper">
         <div class="banner-wrapper-content">
           <h1>
-            Learn. Challenge Yourself. Advance
+            MIT MicroMasters
           </h1>
           <h2>
             Top ranked programs from the #1 university in the world now delivered online. Advance your career with a fast-track master’s degree from MIT.
```

Comment:
@roberthouse54 wanted to have `h1` and `<title>` the same. Maybe we should discuss this.